src/build-order-actions.js

```diff
@@ -109,6 +110,38 @@ const buildOrderMethods = {
     return actions
       .filter(action => enabledActions.indexOf(action.action) >= 0)
   },
+
+  shippingInfo: (order, existingOrder) => {
+    const sync = new OrderSync()
+    const enabledActions = [
+      'addDelivery',
+      'addParcelToDelivery',
+    ]
+
+    /* eslint-disable no-param-reassign */
+    order = _.defaults(order, {
```

Comment:
Why are you reassigning the parameter? You could have used a new variable entirely.

```js
    const modOrder = { shippingInfo: { deliveries: [] }, ...order }
    const modExistOrder = { shippingInfo: { deliveries: [] }, ...existingOrder }
```
And then you wouldn't need underscore here ;)